straxen/plugins/event_patternfit.py

```diff
@@ -133,6 +142,11 @@ def compute(self, events):
         positions = np.vstack([events['x'], events['y'], events['z']]).T
         aft_prob = self.s1_aft_map(positions)
         
+        alt_drift_time = events['s2_center_time']-events['alt_s1_center_time']
+        alt_z = -self.electron_drift_velocity*(alt_drift_time-self.electron_drift_time_gate)
+        alt_position = np.vstack([events['x'], events['y'], alt_z]).T     
```

Comment:
Shouldn't you have alt_* here?

Reply:
Actually no, it has assumed that the alternative s1 happens in the same corrected (x;y) position of the main s1. Furthermore, we don't have corrected alt_x and alt_y.